src/main/java/org/jboss/JiraIssueLotteryCommand.java

```diff
@@ -42,7 +40,8 @@ public void setup() {
     }
 
     private JiraEndpoint setupJiraEndpoint(JiraConfiguration jiraConfiguration) {
-        JiraEndpoint jiraEndpoint = new JiraEndpoint("issues.redhat.com", new JiraComponent(camelContext), jiraConfiguration);
+        JiraEndpoint jiraEndpoint = jiraEndpointProducer.getEndpoint("issues.redhat.com",
```

Comment:
should issues.redhat.com be configurable?

Reply:
This was not decided yet, whether it should be in the config file or application.properties (Also to regards with multiple jiras, for example hibernate)